NEWS.md

```diff
@@ -5,6 +5,28 @@ This file contains the main features as well as overviews of specific
 bug fixes (and other actions) for each version of Libfabric since
 version 1.0.
 
+v1.5.0, TBD
+===========
+
+## PSM provider notes
+
+- Improve the name server functionality and move to the utility code
+- Handle updated mr_mode definitions
+- Add support of 32 and 64 bit atomic values
+
+## PSM2 provider notes
+
+- Add option to adjust the locking level
+- Improve the name server functionality and move to the utility code
+- Add support for string address format
+- Add an environment vaiable for message inject size
+- Handle FI_DISCARD in tagged receive functions
+- Handle updated mr_mode definitions
+- Add support for scalable endpoint
+- Add support of 32 and 64 bit atomic values
+- Add FI_SOURCE_ERR to the supported caps
+- Improve the method of checking device existence
+
```

Comment:
I would rather only have the 1.5.0 release be in the final notes, and not keep the alpha notes (below) separate.  Can you merge the above notes with the alpha section?  I will update the release and date to correspond to 1.5.0rc1, and move that forward, so that the news only ends up with 1.5.0.